sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -3,71 +3,12 @@
-        BlobBlobClientCopyFromUrlOptions, BlobBlobClientCreateSnapshotOptions,
-        BlobBlobClientDeleteImmutabilityPolicyOptions, BlobBlobClientDeleteOptions,
-        BlobBlobClientDownloadOptions, BlobBlobClientGetAccountInfoOptions,
-        BlobBlobClientGetPropertiesOptions, BlobBlobClientGetTagsOptions,
-        BlobBlobClientQueryOptions, BlobBlobClientReleaseLeaseOptions,
-        BlobBlobClientRenewLeaseOptions, BlobBlobClientSetExpiryOptions,
-        BlobBlobClientSetHttpHeadersOptions, BlobBlobClientSetImmutabilityPolicyOptions,
-        BlobBlobClientSetLegalHoldOptions, BlobBlobClientSetMetadataOptions,
-        BlobBlobClientSetTagsOptions, BlobBlobClientSetTierOptions,
-        BlobBlobClientStartCopyFromUrlOptions, BlobBlobClientUndeleteOptions,
-        BlobBlockBlobClientCommitBlockListOptions, BlobBlockBlobClientGetBlockListOptions,
-        BlobBlockBlobClientPutBlobFromUrlOptions, BlobBlockBlobClientStageBlockFromUrlOptions,
-        BlobBlockBlobClientStageBlockOptions, BlobBlockBlobClientUploadOptions,
-        BlobContainerClientAcquireLeaseOptions, BlobContainerClientBreakLeaseOptions,
-        BlobContainerClientChangeLeaseOptions, BlobContainerClientCreateOptions,
-        BlobContainerClientDeleteOptions, BlobContainerClientFilterBlobsOptions,
-        BlobContainerClientGetAccessPolicyOptions, BlobContainerClientGetAccountInfoOptions,
-        BlobContainerClientGetPropertiesOptions, BlobContainerClientReleaseLeaseOptions,
-        BlobContainerClientRenameOptions, BlobContainerClientRenewLeaseOptions,
-        BlobContainerClientRestoreOptions, BlobContainerClientSetAccessPolicyOptions,
-        BlobContainerClientSetMetadataOptions, BlobContainerClientSubmitBatchOptions,
-        BlobPageBlobClientClearPagesOptions, BlobPageBlobClientCopyIncrementalOptions,
-        BlobPageBlobClientCreateOptions, BlobPageBlobClientResizeOptions,
-        BlobPageBlobClientUpdateSequenceNumberOptions, BlobPageBlobClientUploadPagesFromUrlOptions,
-        BlobPageBlobClientUploadPagesOptions, BlobServiceClientFilterBlobsOptions,
-        BlobServiceClientGetAccountInfoOptions, BlobServiceClientGetPropertiesOptions,
-        BlobServiceClientGetStatisticsOptions, BlobServiceClientGetUserDelegationKeyOptions,
-        BlobServiceClientSetPropertiesOptions, BlobServiceClientSubmitBatchOptions,
-    };
-    pub use crate::generated::enums::*;
-    pub use crate::generated::models::*;
-
-    mod blob_properties;
-    pub use blob_properties::BlobProperties;
-
-    mod container_properties;
-    pub use container_properties::ContainerProperties;
-}
-
-pub use crate::generated::clients::{BlobClient, BlobClientOptions};
-// END GENERATED CODE
-
-pub(crate) mod pipeline;
+// Moved from src/generated/mod.rs to surface these specific Options bags to the crate root
+// pub use crate::generated::clients::{
+//     BlobClientOptions, BlobContainerClientOptions, BlobServiceClientOptions,
+// };
```

Comment:
Remove commented code?

Reply:
This will be removed once I merge in the POC changes for the exports.